src/svg/opentable.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+   width="100"
+   height="100"
+   viewBox="0 0 26.458333 26.458334"
```

Comment:
It looks like every icons we have are using a `viewBox` of "0 0 24 24" with no `width` and `height` defined.